src/coreclr/vm/field.h

```diff
@@ -483,22 +483,7 @@ class FieldDesc
         return *(OBJECTREF *)GetCurrentStaticAddress();
     }
 
-    VOID SetStaticOBJECTREF(OBJECTREF objRef)
-    {
-        CONTRACTL
-        {
-            THROWS;
-            GC_TRIGGERS;
-            MODE_COOPERATIVE;
-            INJECT_FAULT(COMPlusThrowOM());
-        }
-        CONTRACTL_END
-
-        GCPROTECT_BEGIN(objRef);
-        OBJECTREF *pObjRef = (OBJECTREF *)GetCurrentStaticAddress();
-        SetObjectReference(pObjRef, objRef);
-        GCPROTECT_END();
-    }
+    VOID SetStaticOBJECTREF(OBJECTREF objRef);
```

Comment:
This is also related to access to FieldDesc on linux x86 in `IsArgumentInRegister`. Otherwise, there are build errors with SetStaticOBJECTREF defined in header, since now "field.h" is included in "common.h"